### DISCUSSION TITLE
Welcome to vscode-cpptools Discussions!

### DISCUSSION THREAD

**Discussion by sean-mcmanus** [microsoft/vscode-cpptools#6800] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by adhithyankrishna** [comment]
after update my windows my vs code were compliling verey slowly also terminal not well before i worked ,and frequently isows pop up like reload .host

**Reply by sean-mcmanus** [reply]
Does your issue repro with the C/C++ extension disabled? Our extension shouldn't have an affect on compiling speed.

**Reply by adhithyankrishna** [reply]
after disabled c/c++ extension still i got same problem ,terminal get too slow to load , if this may be my pc cannot handle windows 10 , recently i update my pc from 7 to 10

**Reply by sean-mcmanus** [reply]
Yes, then it's not an issue with our extension. I'm not sure which terminal you're using, but if you think VS Code has an issue you can try https://github.com/microsoft/vscode/issues .
